diff --git a/lib/get-write-flag.js b/lib/get-write-flag.js
index e86959996623c897ee8440acc34441e46bd44678..35f547982001b132751eff64d39cc25caa1ae6b9 100644
--- a/lib/get-write-flag.js
+++ b/lib/get-write-flag.js
@@ -7,7 +7,7 @@
 // it can be a big boost on Windows platforms.
 // Only supported in Node v12.9.0 and above.
 const platform = process.env.__FAKE_PLATFORM__ || process.platform
-const isWindows = platform === 'win32'
+const isWindows = typeof Bun !== "undefined" ? false : platform === 'win32'
 const fs = global.__FAKE_TESTING_FS__ || require('fs')
 
 /* istanbul ignore next */
